.github/actions/build-vars/action.yml

```diff
@@ -0,0 +1,51 @@
+# action.yml
+name: 'Build Variables'
+description: 'Variables shared by build workflows'
+outputs:
+  env_name:
+    value: ${{ steps.build_result.outputs.env_name }}
+  version:
+    value: ${{ steps.version_result.outputs.version }}
+
+runs:
+  using: "composite"
+  steps:
+
+    - name: Set Build Environment - DEV
```

Comment:
FWIW I think these will change based on the team's discussion of how they want to do development. That is, there may only be `DEV` and `TEST` for now. (And maybe not even `DEV`?)